ATS/db/seeds.rb

```diff
@@ -35,90 +35,112 @@
-	{name: "Forensic Psychology", department_id: 9, degree_type: 2},
-	{name: "Sports Psychology", department_id: 9, degree_type: 2},
-	{name: "Health and Rehabilitation Psychology", department_id: 9, degree_type: 2},
-	{name: "Health and Rehabilitation Psychology (pre-OT)", department_id: 9, degree_type: 2},
-	{name: "Accounting", department_id: 12, degree_type: 2},
-	{name: "Business Administration", department_id: 12, degree_type: 2},
-	{name: "Management", department_id: 12, degree_type: 2},
-	{name: "Management Information Systems", department_id: 12, degree_type: 2},		
-	{name: "Bachelors of Business Administration", department_id: 12, degree_type: 1},
-	{name: "Masters of Business Administration", department_id: 12, degree_type: 16},
-	{name: "B.S. in Business", department_id: 12, degree_type: 12},
-	{name: "M.S. in Business", department_id: 12, degree_type: 17},
-	{name: "Human Resource Management", department_id: 12, degree_type: 2},
-	{name: "Bibilical Studies", department_id: 10, degree_type: 3},
-	{name: "Bibilical Studies/MA", department_id: 10, degree_type: 7},
-	{name: "Christian Ministry", department_id: 10, degree_type: 7},
-	{name: "Christian Counseling and Family Ministry", department_id: 10, degree_type: 7},
-	{name: "Master of Philosphy", department_id: 10, degree_type: 7},
-	{name: "Bibilical Studies/PhD", department_id: 10, degree_type: 10},
-	{name: "Collaborative Education", department_id: 13, degree_type: 2},
-	{name: "Early Childhood Education", department_id: 13, degree_type: 2},
-	{name: "Elementary Education", department_id: 13, degree_type: 2},
-	{name: "Counseling", department_id: 13, degree_type: 8},
-	{name: "School Counseling", department_id: 13, degree_type: 8},
-	{name: "Teaching", department_id: 13, degree_type: 7},
-	{name: "M.Ed.", department_id: 13, degree_type: 15},
+BS = DegreeType.find_by_name("Bachelors of Science")
+BA = DegreeType.find_by_name("Bachelors of Arts")
+ASdeg = DegreeType.find_by_name("Associate of Science")
+MS = DegreeType.find_by_name("Masters of Science")
+MA = DegreeType.find_by_name("Masters of Arts")
+PHD = DegreeType.find_by_name("PhD")
+Minor = DegreeType.find_by_name("Minor")
+CSdept = Department.find_by_name("Computer Science")
+CJdept = Department.find_by_name("Criminal Justice/Legal Studies")
+Engdept = Department.find_by_name("English")
+Humdept = Department.find_by_name("Humanities")
+Musdept = Department.find_by_name("Music")
+Busdept = Department.find_by_name("Business")
+Edudept = Department.find_by_name("Education")
+Bibdept = Department.find_by_name("Bibilical Studies")
+Kindept = Department.find_by_name("Kinesiology and Exercise Science")
+Scidept = Department.find_by_name("Natural and Physical Sciences")
+Socdept = Department.find_by_name("Social and Behavioral Sciences")
+degrees = [{name: "Computer Science", department_id: CSdept, degree_type: BS},
+	{name: "Computer Information Science", department_id: CSdept, degree_type: BS},
```

Comment:
@sah0017 Same situation here. CSdept.id should return the integer that is needed in the hash. BS.id should work in the same fashion for degree_type. Byebug does work inside the seed file so it may be a good idea to use that and inspect these objects you are creating.